src/xrGame/ui/UIListItemServer.cpp

```diff
@@ -99,22 +99,22 @@ void CUIListItemServer::InitItemServer(LIST_SRV_ITEM& params)
 }
 
 #include "string_table.h"
-u32 cut_string_by_length(CGameFont* pFont, LPCSTR src, LPSTR dst, u32 dst_size, float length);
+u32 CutStringByLength(CGameFont* font, LPCSTR src, LPSTR dst, u32 dstSize, float length);
```

Comment:
Not related to gamepads, not even related to input system. Please send it as another pull request for MP map weather list fix.

Reply:
No, I can't. Because this function is the part of xr input associated code refactoring. Watch `xrGame/ui/UIEditKeyBind.cpp` and you'll see where is defenision.